test/collatz_test.js

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai'
+import collatz from '../src/collatz.js'
+
+describe('collatz()', function(){
+  it('should be a function', function(){
+    expect(collatz).to.be.a('function')
+  })
+
+  it('should return the collatz conjecture of an integer', function(){
```

Comment:
I would separate these three expects out into three separate it()s. When there are lots of expects in one, it gets harder to figure out which test is failing. It might seem a little pointless, but getting into good habits will be beneficial down the line <3